src/resources/shipment.js

```diff
@@ -169,6 +169,30 @@ export default (api) =>
       return this.rpc('insure', { amount });
     }
 
+    /**
+     * Generate a form for a shipment.
+     * @param {string} formType
+     * @param {object} formOptions
+     * @returns {this}
+     */
+    async generateForm(formType, formOptions) {
+      const _formOptions = formOptions || {};
```

Comment:
```suggestion
    async generateForm(formType, formOptions = {}) {
      const _formOptions = formOptions || {};
```

I think you can make `formOptions` as an optional parameter by doing this way